modules/editorial-metadata/editorial-metadata.php

```diff
@@ -413,32 +405,29 @@ public static function update_editorial_metadata_term( int $term_id, array $args
 	 * @return bool $result Whether or not the term was deleted
 	 */
 	public static function delete_editorial_metadata_term( int $term_id ): bool {
-		$term          = self::get_editorial_metadata_term_by( 'id', $term_id );
-		$post_meta_key = self::get_postmeta_key( $term );
+		$term = self::get_editorial_metadata_term_by( 'id', $term_id );
+		if ( is_wp_error( $term ) ) {
+			return $term;
+		} else if ( ! $term ) {
+			return new WP_Error( 'invalid', __( "Editorial metadata term doesn't exist.", 'vip-workflow' ) );
+		}
+
+		// Delete the post meta for the term
+		$post_meta_key = self::get_postmeta_key( $term->meta[ self::METADATA_TYPE_KEY ], $term_id );
 		delete_post_meta_by_key( $post_meta_key );
 
-		$result = wp_delete_term( $term_id, self::METADATA_TAXONOMY );
+		delete_term_meta( $term_id, self::METADATA_TYPE_KEY );
```

Comment:
Didn't bother with the result as the core term is being killed. This stopped deletes when it shouldn't have